src/api/API.ts

```diff
@@ -13,7 +13,7 @@ export class API {
         const currentUserId = await keytar.getPassword(KeytarKeys.service, KeytarKeys.userId);
         if (accessToken && currentUserId) {
           const url = `https://api.twitch.tv/helix/users/follows?from_id=${twitchUserId}&to_id=${currentUserId}`;
-          const res = await fetch(url, { headers: { 'Authorization': `Bearer ${accessToken}` } });
+          const res = await fetch(url, { headers: { 'Authorization': `Bearer ${accessToken}`, 'client-id': 'ts9wowek7hj9yw0q7gmg27c29i6etn' } });
```

Comment:
Would it not be better to make the client is into a const? Rather declaring it 3 times in the code?

Reply:
I don't disagree.  Feel free to submit a PR. 😄